site/_docs/howto.md

```diff
@@ -841,6 +841,8 @@ your key to the keyservers used by Nexus, see above.
   that the `META-INF` directory contains `LICENSE`,
   `NOTICE`
 * Check PGP, per [this](https://httpd.apache.org/dev/verification.html)
+* Check that Proj4J is not among the api and implementation dependencies,
```

Comment:
We could enforce this with a new CI job as well. Not necessary to do it now but maybe worth considering as a follow-up.

Reply:
Yes, this is a good idea. Would adding something like `if ./gradlew dependencies | grep -q proj4j; then exit 1; else exit 0; fi` in a workflow be sufficient?

Reply:
I think proj4j are going to fix the licensing issue on their end. I think a CI job would be overkill given that this fix will be short-lived.

I suggest adding a 'Add Proj4j as a dependency' Jira case, to be fixed in 1.34, and revisit then.

Reply:
I've made some further remarks in the jira case, and +1 when those are fixed.